### DISCUSSION TITLE
New Release ggb-v0.4.2-Community-Extension-GM-All

### DISCUSSION THREAD

**Discussion by cmalf** [cmalf/getgrass-bot-js#18] [General]
# ✨ New Release Update

# 🌱 getgrass Bot - Community Extension version (GM ALL) (Multi & Normal Version)

# Release : [getgrass-bot-v0.4.2-GM-ALL](https://github.com/cmalf/getgrass-bot-js/releases/tag/getgrass-bot-v0.4.2-GM-ALL)
# Telegram Channel : [@getgrassbotjs](https://t.me/getgrassbotjs)

## 🆕 Whats New

-  ***No longer used :***
      * > wss://proxy2.wynd.network:4444/
      * > wss://proxy2.wynd.network:4650/
- ***using new trick: because most of the proxies are blocked by `proxy2.wynd.network`***
-  Improved `error handling` fixes
- Script fixes including bugs found in previous versions.
- Lighter script
- Script improvements to support all platforms (macos, windows, linux, android-termux, ios-ISH-shell) without having to manually set it to fix errors.

## 🌆 Screenshot
- MultiAccounts Script

<img src="https://github.com/user-attachments/assets/a81124b0-9650-4083-bba0-070b8f2ce2b1" widht=580 height=480 >

<hr>

- Single Account Script (Normal)

<img src="https://github.com/user-attachments/assets/26cf1121-24e8-4616-b11a-6a7f7436179d" widht=580 height=480 >

**Comment by latifangren** [comment]
check telegram pls, im already send screenshoot

**Reply by cmalf** [reply]
> check telegram pls, im already send screenshoot

I have replied on Telegram.

**Reply by 0xsown** [reply]
Hi bro, still only support `socks5` right ?

**Reply by 0xsown** [reply]
@cmalf

**Reply by latifangren** [reply]
which proxy format did u use?

**Comment by prodesk-1** [comment]
Not working :( Auth false

**Reply by latifangren** [reply]
what proxy format did u use?

**Reply by prodesk-1** [reply]
socks5://username:password@ip:port
Proxies from proxies.fo ISP AT&T

**Reply by prodesk-1** [reply]
Received Message: {"id":"oAhhQEA1ExDFWAUdjv7GD","action":"HTTP_REQUEST","data":{"url":"https://api.getgrass.io/e65c798e44045cf3241e5bdc52aff12c/owtqbwqb5P4lpL3p","method":"GET","headers":{"Accept":"*/*","Host":"api.getgrass.io","User-Agent":"wynd.network/3.0.1"},"body":null,"authenticated":false}}

**Reply by cmalf** [reply]
> Received Message: {"id":"oAhhQEA1ExDFWAUdjv7GD","action":"HTTP_REQUEST","data":{"url":"[https://api.getgrass.io/e65c798e44045cf3241e5bdc52aff12c/owtqbwqb5P4lpL3p","method":"GET","headers":{"Accept":"*/*","Host":"api.getgrass.io","User-Agent":"wynd.network/3.0.1"},"body":null,"authenticated":false}}](https://api.getgrass.io/e65c798e44045cf3241e5bdc52aff12c/owtqbwqb5P4lpL3p%22,%22method%22:%22GET%22,%22headers%22:%7B%22Accept%22:%22*/*%22,%22Host%22:%22api.getgrass.io%22,%22User-Agent%22:%22wynd.network/3.0.1%22%7D,%22body%22:null,%22authenticated%22:false%7D%7D)

no problem because this is original from grass server.

**Reply by prodesk-1** [reply]
Still mining ?

**Reply by cmalf** [reply]
> Still mining ?

yes

**Reply by prodesk-1** [reply]
Thanks brother :)

**Reply by latifangren** [reply]
im working well check dashboard and got bunch of new device ip connected

**Reply by jacktruong1** [reply]
why i dont copy pass script ? how i can copy pass

**Comment by 20dec** [comment]
How about desktop app bro?

**Reply by cmalf** [reply]
> How about desktop app bro?

[Information update, about getgrass desktop version (script) 📢❗🚨](https://github.com/cmalf/getgrass-bot-js/discussions/16)

**Comment by cryptoleveling** [comment]
The previous version was better

**Comment by vladislaw236** [comment]
If i use Multi script, it log in from 1 ip to all my accounts?

**Comment by crkerx** [comment]
![image](https://github.com/user-attachments/assets/5fbed9a3-d1ce-44df-9d75-499930489588)
![image](https://github.com/user-attachments/assets/37edf2a6-d2ea-4593-aae5-0a67c2c2d241)
![image](https://github.com/user-attachments/assets/d5a76a6b-f670-426c-a4fd-0937b6f23ce2)

i'm using ISP proxy from proxies.fo provider, when i'm check proxies by your tool (proxy-ip-checker-php), almost of the proxies are alive. However, when i'm run this script, after few minutes, it started reporting an error related to the proxy (according to the image I attached).

**Reply by prodesk-1** [reply]
Use socks protocol
Proxy Format: socks5://username:password@ip:port

Everything will run smooth.

**Reply by gimbung** [reply]
i have same issue , i already use socks5

**Reply by baby5226** [reply]
The same is true for me. It has been normal for 2 days, and just now I suddenly got the same error as in the photo.

**Reply by 0xsown** [reply]
this tool support only socks5, http proxy unsupported

**Reply by baby5226** [reply]
> 對我來說也是如此。已經正常2天了，剛才突然出現了跟照片一樣的錯誤。

I have the same problem today. I have been using socks5 from before to now, but suddenly it stopped working on the night of 12/30.

**Comment by vladislaw236** [comment]
TypeError: Cannot read properties of undefined (reading 'map')
    at _0x428a33 (/root/grass042/MultiN042/index.js:1:285977)
    at Object.Nmoxw (/root/grass042/MultiN042/index.js:1:118168)
    at Object.GYbix (/root/grass042/MultiN042/index.js:1:174791)
    at /root/grass042/MultiN042/index.js:1:180053
    at Interface._onLine (node:readline:489:5)
    at Interface._line (node:readline:868:8)
    at Interface._ttyWrite (node:readline:1220:14)
    at ReadStream.onkeypress (node:readline:291:10)
    at ReadStream.emit (node:events:513:28)
    at emitKeys (node:internal/readline/utils:357:14)
    
    
    What to do with that problem? Early in Normal i havent that
    
    This is new 0.4.2

**Comment by azisabdul756** [comment]
Apakah ini menambang atau terhenti?
![grass1](https://github.com/user-attachments/assets/1fa2afbe-2573-43a5-add7-8efc66ae8ae4)

**Reply by Faisalazmi** [reply]
setau saya jika ada PONG itu sedang menambang

**Comment by cmalf** [comment]
I apologize for any delays in responding to your messages. I've been trying to enjoy some time off work, but it seems I'm still glued to my computer.

With New Years just a day away, my son was excited to go somewhere to celebrate, and my wife has gently reminded me that it's time to step away from the screen. I want to be present for my family during this special time.

I'll get back to you as soon as I can. Wishing you a happy and safe New Year! 🎇🎉

**Reply by tuantep** [reply]
HPNY 2025!!!

**Reply by vebri223** [reply]
plis renew

**Comment by rahmap** [comment]
@cmalf can i use IPv6 Residential ?
![image](https://github.com/user-attachments/assets/2d5bf5f5-906a-41e9-a4a1-2cb15709e9cc)

**Reply by 20dec** [reply]
no bro

**Reply by rahmap** [reply]
did you ever try?

**Reply by 20dec** [reply]
Yes

**Reply by rahmap** [reply]
oke bro, thanks

**Comment by cmalf** [comment]
@ Everyone
![Screenshot 2025-01-02 at 19 19 33](https://github.com/user-attachments/assets/5955293c-c22c-452e-8076-b408c4f2906a)
[v0.4.2-Extended](https://github.com/cmalf/getgrass-bot-js/releases/tag/getgrass-bot-v0.4.2-GM-ALL)

**Comment by vebri223** [comment]
i already do following the instruction to get password sir, can you give me the password?
![6316831010838397602](https://github.com/user-attachments/assets/28412e6b-6633-4c73-b045-c487ac2aa774)

**Comment by cmalf** [comment]
![Screenshot 2025-01-03 at 23 06 47](https://github.com/user-attachments/assets/58755d18-b4c7-4878-8003-0061adabb5ee)

**Reply by VEBRI2353** [reply]
Still ming or no bro in dashboard still not conect

**Reply by VEBRI2353** [reply]
I buy proxy residential and sticky just same not conect in dashboard grass pliss reply sir🙏

**Reply by cmalf** [reply]
> I buy proxy residential and sticky just same not conect in dashboard grass pliss reply sir🙏

which version of the script are you using.

**Reply by VEBRI2353** [reply]
ggb-v0.4.2-GM-ALL with extended

**Reply by cmalf** [reply]
> ggb-v0.4.2-GM-ALL with extended

The script still works fine, and the proxy is detected in the dashboard. Does your proxy work stably in the script?

**Reply by VEBRI2353** [reply]
no but sometimes connected sometimes not but in grass not connected

**Reply by VEBRI2353** [reply]
I use proxy residential and I swipe to  sticky but same problem 🙏

**Reply by cmalf** [reply]
> I use proxy residential and I swipe to sticky but same problem 🙏

are you using season 1 account? if yes, maybe your account is detected as sybil account

**Reply by VEBRI2353** [reply]
No I use season 2

**Reply by VEBRI2353** [reply]
![Screenshot_20250104-013741_Chrome~2](https://github.com/user-attachments/assets/e82bd3f2-e0f2-4e80-b8b3-b363d7aa08c3)

**Reply by VEBRI2353** [reply]
Pliss reply  why not working

**Comment by cmalf** [comment]
![Screenshot 2025-01-08 at 09 13 39](https://github.com/user-attachments/assets/05ddde70-5cdc-4103-bbca-7236aab2b41c)
- v0.4.1-Login-Proxy : https://shorturl.at/nM8mU
- v0.4.2-Login-Proxy : https://shorturl.at/C5hAX

**Reply by VEBRI2353** [reply]
Can I use to normal version?

**Comment by nassirok** [comment]
please where can find Free proxy ?

**Comment by toanbk** [comment]
I cannot find package.json on zip file, anyone can send it for me ?

**Reply by MHendriF** [reply]
bro you can find in folder [ggb-v0.4.2-GM-ALL.zip](https://github.com/cmalf/getgrass-bot-js/releases/download/getgrass-bot-v0.4.2-GM-ALL/ggb-v0.4.2-GM-ALL.zip)

**Reply by VEBRI2353** [reply]
Script still work?

**Reply by cmalf** [reply]
> Script still work?

yes.

**Comment by prodesk-1** [comment]
Consuming lots of bandwidth from 2-3 days :( 25 GB plan over in 3 days :( BTW i am using 3000 proxies :)

**Reply by cmalf** [reply]
yes of course, because bandwidth is wasted on overloaded servers. I often recommend running only 50 - 100 proxies per account, for now.

**Comment by vebri223** [comment]
![Screenshot_20250204-150130_Microsoft Remote Desktop](https://github.com/user-attachments/assets/96b0d712-4a64-4454-b8e1-f66ae46e2a87)

**Reply by vebri223** [reply]
Why is the uptime over 100 days? Does it mean we are using the same proxy as others? If we are using a proxy, would you recommend changing it?
